go.mod

```diff
@@ -3,15 +3,26 @@ module github.com/crypto-org-chain/cronos
 go 1.16
 
 require (
+	github.com/Masterminds/goutils v1.1.1 // indirect
```

Comment:
These dependencies are generated by the protocgen script, I think it's preferred to not included?

Reply:
yeah its auto generated. Do you mean I have to manually remove them?

Reply:
I usually do a `git checkout -- go.mod go.sum` after run the proto-gen script.

Reply:
Because they are not really dependencies of the project itself.

Reply:
got it! done